tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -321,6 +324,68 @@ func TestGlobalCheckAndReport(t *testing.T) {
 	}
 }
 
+func TestTcpMetrics(t *testing.T) {
+	if err := replaceRouteRule(tcpDbRule); err != nil {
```

Comment:
ah nice, this is the test mentioned in the other PR, sorry somehow I didn't see that PR until now
(don't hesitate to ping me in the future if I don't comment on a PR before it gets merged)